### ISSUE TITLE
ValueError at /api/manage-token/sw.js invalid literal for int() with base 10: 'sw'

### ISSUE THREAD

**Issue by jw35** [SmartCambridge/tfc_web#342]
```
Internal Server Error: /api/manage-token/sw.js

ValueError at /api/manage-token/sw.js
invalid literal for int() with base 10: 'sw'

Request Method: GET
Request URL: https://smartcambridge.org/api/manage-token/sw.js

Traceback:  

File "/home/tfc_prod/tfc_web_venv/lib/python3.5/site-packages/django/core/handlers/exception.py" in inner
 41.             response = get_response(request)

File "/home/tfc_prod/tfc_web_venv/lib/python3.5/site-packages/django/core/handlers/base.py" in _legacy_get_response
 249.             response = self._get_response(request)

File "/home/tfc_prod/tfc_web_venv/lib/python3.5/site-packages/django/core/handlers/base.py" in _get_response
 187.                 response = self.process_exception_by_middleware(e, request)

File "/home/tfc_prod/tfc_web_venv/lib/python3.5/site-packages/django/core/handlers/base.py" in _get_response
 185.                 response = wrapped_callback(request, *callback_args, **callback_kwargs)

File "/home/tfc_prod/tfc_web_venv/lib/python3.5/site-packages/django/contrib/auth/decorators.py" in _wrapped_view
 23.                 return view_func(request, *args, **kwargs)

File "/home/tfc_prod/tfc_web_venv/lib/python3.5/site-packages/django/contrib/auth/decorators.py" in _wrapped_view
 23.                 return view_func(request, *args, **kwargs)

File "/home/tfc_prod/tfc_web_venv/lib/python3.5/site-packages/django/contrib/auth/decorators.py" in _wrapped_view
 23.                 return view_func(request, *args, **kwargs)

File "/home/tfc_prod/tfc_web/tfc_web/authmultitoken/views.py" in manage_token
 108.             id=token_id)

File "/home/tfc_prod/tfc_web_venv/lib/python3.5/site-packages/django/db/models/manager.py" in manager_method
 85.                 return getattr(self.get_queryset(), name)(*args, **kwargs)

File "/home/tfc_prod/tfc_web_venv/lib/python3.5/site-packages/django/db/models/query.py" in get
 371.         clone = self.filter(*args, **kwargs)

File "/home/tfc_prod/tfc_web_venv/lib/python3.5/site-packages/django/db/models/query.py" in filter
 787.         return self._filter_or_exclude(False, *args, **kwargs)

File "/home/tfc_prod/tfc_web_venv/lib/python3.5/site-packages/django/db/models/query.py" in _filter_or_exclude
 805.             clone.query.add_q(Q(*args, **kwargs))

File "/home/tfc_prod/tfc_web_venv/lib/python3.5/site-packages/django/db/models/sql/query.py" in add_q
 1250.         clause, _ = self._add_q(q_object, self.used_aliases)

File "/home/tfc_prod/tfc_web_venv/lib/python3.5/site-packages/django/db/models/sql/query.py" in _add_q
 1276.                     allow_joins=allow_joins, split_subq=split_subq,

File "/home/tfc_prod/tfc_web_venv/lib/python3.5/site-packages/django/db/models/sql/query.py" in build_filter
 1210.             condition = self.build_lookup(lookups, col, value)

File "/home/tfc_prod/tfc_web_venv/lib/python3.5/site-packages/django/db/models/sql/query.py" in build_lookup
 1104.                 return final_lookup(lhs, rhs)

File "/home/tfc_prod/tfc_web_venv/lib/python3.5/site-packages/django/db/models/lookups.py" in __init__
 24.         self.rhs = self.get_prep_lookup()

File "/home/tfc_prod/tfc_web_venv/lib/python3.5/site-packages/django/db/models/lookups.py" in get_prep_lookup
 74.             return self.lhs.output_field.get_prep_value(self.rhs)

File "/home/tfc_prod/tfc_web_venv/lib/python3.5/site-packages/django/db/models/fields/__init__.py" in get_prep_value
 966.         return int(value)

Exception Type: ValueError at /api/manage-token/sw.js
Exception Value: invalid literal for int() with base 10: 'sw'
Request information:
USER: tom_bardsley

GET: No GET data

POST: No POST data

FILES: No FILES data
```

**Comment by jw35** [comment]
This is probably a side effect of #337

**Comment by jw35** [comment]
It doesn't seem to be service affecting.

**Comment by jw35** [comment]
Fixed by #344
